en/flight_modes_rover/mecanum.md

```diff
@@ -0,0 +1,128 @@
+# Drive Modes (Mecanum Rover)
+
+Flight modes (or more accurately "Drive modes" for ground vehicles) provide autopilot support to make it easier to manually drive the vehicle or to execute autonomous missions.
+
+This section outlines all supported drive modes for mecanum rovers.
+
+For information on mapping RC control switches to specific modes see: [Basic Configuration > Flight Modes](../config/flight_mode.md).
+
+::: warning
+Selecting any other mode than those listed below will either stop the rover or can lead to undefined behaviour.
+:::
+
+## Manual Modes
+
+Manual modes require stick inputs from the user to drive the vehicle.
+
+![Manual Controls](../../assets/airframes/rover/flight_modes/manual_controls_mecanum.png)
+
+The manual modes listed below provide increasing levels of autopilot support:
+
+| Mode                           | Forward/Lateral speed                                                       | Yaw rate                                                                                                                                                                                          | Required measurements                              |
+| ------------------------------ | --------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | -------------------------------------------------- |
+| [Manual](#manual-mode)         | Directly map stick input to motor commands.                                 | Directly map stick input to motor commands.                                                                                                                                                       | None.                                              |
+| [Acro](#acro-mode)             | Directly map stick input to motor commands.                                 | Stick input creates a yaw rate setpoint for the control system to regulate.                                                                                                                       | Yaw rate.                                          |
+| [Stabilized](#stabilized-mode) | Directly map stick input to motor commands.                                 | Stick input creates a yaw rate setpoint for the control system to regulate. If this setpoint is zero (stick is centered) the control system will maintain the current yaw (heading) of the rover. | Yaw rate and yaw.                                  |
+| [Position](#position-mode)     | Stick input creates a velocity setpoint for the control system to regulate. | Stick input creates a yaw rate setpoint for the control system to regulate. If this setpoint is zero (stick is centered) the control system will keep the rover driving in a straight line.       | Yaw rate, yaw, velocity and global position (GPS). |
```

Comment:
I like what you did in the comparison for one of the modes below

> Compared to [Acro mode](#acro-mode), this mode is much better at driving in a straight line as it can more effectively reject disturbances.

This might be worth grafting into this table. From my perspective I'm interested in "what have you done for me lately" in as succinct a way as possible.

Does that make sense? I guess it depends on what is important to your audience.

Reply:
I added more info on the differences of the manual modes in https://github.com/PX4/PX4-user_guide/pull/3411/commits/ad2625b2c3d29b661018b2b47a4dc7dfa98ecc84.
I feel like adding this to the table will overload it, but I can rethink it if you disagree (Maybe _only_ have that information instead of what is in the table now?).

Reply:
For discussion I have added a table up the top with this information. Can you check that.

I agree that this might overload things, so I have hidden the other information in a details box. This might still be too "busy" so we can remove/move if you think better.

See https://github.com/PX4/PX4-user_guide/pull/3411/commits/3ce9cb14146808669458c4ed3855d0aa6298e8eb

Reply:
I like this idea. I made some minor corrections to it in https://github.com/PX4/PX4-user_guide/pull/3411/commits/2ce81ca07acfb95edd4f04916893426bad641d28